yt_dlp/extractor/common.py

```diff
@@ -994,6 +995,10 @@ def _guess_encoding_from_content(content_type, webpage_bytes):
 
         return encoding
 
+    @staticmethod
+    def _is_jwt_token_expired(token):
+        return jwt_decode_hs256(token)['exp'] - time.time() < 300
+
```

Comment:
IMO this belongs in `utils`

I think the function name should be `jwt_is_expired`:
- `jwt_token` is redundant (jwt == json web token)
- `if jwt_is_expired()` reads better than `if is_jwt_expired()`

We also need to add some params to avoid changing the extractors' current behavior and for future-proofing (since utils are public API):

```py
def jwt_is_expired(token, buffer=300, key='exp'):
    return jwt_decode_hs256(token)[key] - time.time() < buffer
```